tests/CommandLine.Tests/Unit/Issue482Tests.cs

```diff
@@ -0,0 +1,200 @@
+                "value pos. 0"
+            };
+            Assert.Equal(expected.Count, helps.Count);
+            int i = 0;
+            foreach (var expect in expected)
+            {
+                Assert.Equal(expect.Trim(), helps[i].Trim());
+                i++;
+            }
+
+            ;
+        }
+
+        [Fact]
+        public void AutoBuild_with_ordering()
+        {
+            string expectedCompany = "Company";
+
+
+            var parser = Parser.Default;
+            var parseResult = parser.ParseArguments<Options_HelpText_Ordering_Verb1, Options_HelpText_Ordering_Verb2>(
+                    new[] { "verb1", "--alpha", "alpaga", "--alpha2", "alala", "--charlie", "charlot" })
+                .WithNotParsed(errors => { throw new InvalidOperationException("Must be parsed."); })
+                .WithParsed(args => {; });
+
+            Comparison<ComparableOption> comparison = HelpText.RequiredThenAlphaComparison;
+
+            string message = HelpText.AutoBuild(parseResult, error =>
+            {
+                throw new InvalidOperationException($"help text build failed. {error.ToString()}");
+            },
+                ex =>
+                    {
+                        return null;
+                    },
+                    false,
+                    80,
+                    comparison);
+            
+
+            string helpMessage = message.ToString();
+            var helps = helpMessage.Split(new[] { '\r', '\n' }, StringSplitOptions.RemoveEmptyEntries).Skip(2).ToList<string>();
+            List<string> expected = new List<string>()
+            {
+                "  -a, --alpha      Required.",
+                "  -b, --alpha2     Required.",
+                "  -c, --bravo",
+                "  -d, --charlie",
+                "-e, --echo",
+                "-f, --foxtrot",
+                "--help           Display this help screen.",
+                "--version        Display version information.",
+                "value pos. 0"
+            };
+            Assert.Equal(expected.Count, helps.Count);
+            int i = 0;
+            foreach (var expect in expected)
+            {
+                Assert.Equal(expect.Trim(), helps[i].Trim());
+                i++;
+            }
+
+            ;
+        }
+
+        [Fact]
+        public void AutoBuild_with_ordering_on_shortName()
+        {
+            string expectedCompany = "Company";
+
+
+            var parser = Parser.Default;
+            var parseResult = parser.ParseArguments<Options_HelpText_Ordering_Verb1, Options_HelpText_Ordering_Verb2>(
+                    new[] { "verb1", "--alpha", "alpaga", "--alpha2", "alala", "--charlie", "charlot" })
+                .WithNotParsed(errors => { throw new InvalidOperationException("Must be parsed."); })
+                .WithParsed(args => {; });
+
+            Comparison<ComparableOption> orderOnShortName = (ComparableOption attr1, ComparableOption attr2) =>
```

Comment:
Remove redundant else and braces, and use StringComparison.Ordinal as done [here](https://github.com/commandlineparser/commandline/pull/484/files#r306971344)